.autover/autover.json

```diff
@@ -127,6 +127,18 @@
         "Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Amazon.Lambda.TestTool.BlazorTester80-pack.csproj",
         "Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Amazon.Lambda.TestTool.BlazorTester90-pack.csproj"
       ]
+    },
+    {
```

Comment:
I was going to a PR soon to remove the VS 2017 and 2019 blueprints. I don't think we should bother putting those here and just have `Amazon.Lambda.Templates` pointing to the VS 2022. If there is a new VS that comes out we can revisit this configuration.

Reply:
Fixed